src/templates/index.tsx

```diff
@@ -246,6 +244,10 @@ const homePageQuery = graphql`
         keynotes: allContentfulSession(filter: {
             sessionType: {
                 in: ["Opening Keynote", "Closing Keynote"]
+            },
+            startTime:{
```

Comment:
StartDate and EndDate are actually times?

Reply:
`startTime` on the session is actually a DateTime